project/templates/configs/springboard.ini

```diff
@@ -0,0 +1,97 @@
+[app:springboardfrontend]
+use = egg:springboard_{{ app_type }}
+unicore.content_repos = unicore_frontend_{{app_type}}_{{country}}
+
+pyramid.default_locale_name = {{default_language}}
+
+featured_languages =
+    {{default_language}}
```

Comment:
By default springboard will show the current language + featured languages. So when the default language is the current language, with this value, only the default language will be displayed in the header. Perhaps we need to put `default_language` plus one or two other available languages here.